### PR TITLE
Fix color space nme

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/core/src/Engines/thinEngine.ts [hunk 1]
```diff
@@ -732,8 +732,12 @@ export class ThinEngine extends AbstractEngine {
                 }
             }
             // take into account the forced state that was provided in options
-            // When the issue in angle/chrome is fixed the flag should be taken into account only when it is explicitly defined
-            this._caps.supportSRGBBuffers = this._caps.supportSRGBBuffers && !!(this._creationOptions && (this._creationOptions as EngineOptions).forceSRGBBufferSupportState);
+            if (this._creationOptions) {
+                const forceSRGBBufferSupportState = (this._creationOptions as EngineOptions).forceSRGBBufferSupportState;
```
Comment (RaananW) [inline packages/dev/core/src/Engines/thinEngine.ts:736]: nit - no need to check for `this._creationOptions`

**Review by Popov72 — APPROVED** [review]

**Review by RaananW — APPROVED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15571/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15571/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15571/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15571/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15571/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15571/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15571/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15571/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15571/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15571/merge/testResults/webgl2playwright/index.html

**Comment by deltakosh** [comment]
@RaananW I would need your expertise. I don't get what is going wrong

**Comment by RaananW** [comment]
> @RaananW I would need your expertise. I don't get what is going wrong

i'm on it. let me check what went wrong here.

**Pull request merged**